src/app/components/FormElements/RegionSelector.tsx

```diff
@@ -1,19 +1,14 @@
 import * as React from 'react';
-import { default as FormFieldWrapper, FormFieldWrapperProps } from './FormFieldWrapper';
-import { FormControl } from 'react-bootstrap';
+import { FormFieldWrapperProps } from './FormFieldWrapper';
 import { REGION } from '../../api';
```

Comment:
I'd probably Pluralize *REGION* -> *REGIONS*

Reply:
That makes sense.